packages/turf-boolean-touches/index.ts

```diff
@@ -0,0 +1,449 @@
+              } 
+              if (booleanPointInPolygon({type: 'Point', coordinates: geom2.coordinates[ii]}, {type:'Polygon', coordinates: geom1.coordinates[0][i]}, {ignoreBoundary: true})) return false;
+            }
+          }
+          return foundTouchingPoint
+        case 'MultiLineString':
+          var foundTouchingPoint = false
+          for (var i = 0; i < geom1.coordinates.length; i++) {
+            for (var ii = 0; ii < geom2.coordinates.length; ii++) {
+              for (var iii = 0; iii < geom2.coordinates[ii].length; iii++) {
+                  if (!foundTouchingPoint) {
+                    if (booleanPointOnLine({type: 'Point', coordinates: geom2.coordinates[ii][iii]}, {type:'LineString', coordinates: geom1.coordinates[i][0]})) foundTouchingPoint = true;
+                  } 
+                  if (booleanPointInPolygon({type: 'Point', coordinates: geom2.coordinates[ii][iii]}, {type:'Polygon', coordinates: [geom1.coordinates[i][0]]}, {ignoreBoundary: true})) return false;
+                }
+            }
+          }
+
+          return foundTouchingPoint
+        case 'Polygon':
+          var foundTouchingPoint = false
+          for (var i = 0; i < geom1.coordinates[0].length; i++) {
+            for (var ii = 0; ii < geom1.coordinates[0][i].length; ii++) {
+              if (!foundTouchingPoint) {
+                if (booleanPointOnLine({type: 'Point', coordinates: geom1.coordinates[0][i][ii]}, {type:'LineString', coordinates: geom2.coordinates[0]})) foundTouchingPoint = true;
+              }
+              if (booleanPointInPolygon({type: 'Point', coordinates: geom1.coordinates[0][i][ii]}, geom2, {ignoreBoundary: true})) return false;
+            }
+          }
+          return foundTouchingPoint
+        case 'MultiPolygon':
+          var foundTouchingPoint = false
+          for (var i = 0; i < geom1.coordinates[0].length; i++) {
+            for (var ii = 0; ii < geom2.coordinates[0].length; ii++) {
+              for (var iii = 0; iii < geom1.coordinates[0].length; iii++) {
+                  if (!foundTouchingPoint) {
+                    if (booleanPointOnLine({type: 'Point', coordinates: geom1.coordinates[0][i][iii]}, {type:'LineString', coordinates: geom2.coordinates[0][ii]})) foundTouchingPoint = true;
+                  }
+                  if (booleanPointInPolygon({type: 'Point', coordinates: geom1.coordinates[0][i][iii]}, {type:'Polygon', coordinates: geom2.coordinates[0][ii]}, {ignoreBoundary: true})) return false;
+              }
+            }
+          }
+          return foundTouchingPoint
+        default:
+            throw new Error('feature2 ' + type2 + ' geometry not supported');
+        }
+    default:
+        throw new Error('feature1 ' + type1 + ' geometry not supported');
+    }
+}
+
+function isPointOnLineEnd(point, line) {
```

Comment:
@rowanwins Instead of including/repeating all of these methods, why don't we start a `@turf/boolean-helpers` that has all of these methods available.

I've noticed the same methods in [`@turf/boolean-contains`](https://github.com/Turfjs/turf/blob/master/packages/turf-boolean-contains/index.ts) & [`@turf/boolean-within`](https://github.com/Turfjs/turf/blob/master/packages/turf-boolean-within/index.ts).

Reply:
yeah good thinking!